src/app-pages/Reports/CorpOfficeReports/SpecialReports/index.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+//import classnames from "classnames";
+import TextSection from '../../../../app-components/TextSection';
+import SearchBox from '../../../../app-containers/SearchBox';
+import Tabs from '../../../../app-components/Tabs';
+
+const containerTextSection = {
+	textAlign: 'center',
+	margin:'1rem 0',
+	padding: 'auto 12rem',
+	width: '100%'
+}
+
+const headerContainerStyle = {
+	backgroundColor: '#cbd5e0'
+}
+
+const TabInfo = {
+  active: 3,
+  hrefs: [
+    '/reports/CorpOfficeReports', 
+    '/reports/ProjectReports',
+    '/reports/WatershedReports',
+    '/reports/DistrictReports'
+  ],
+  titles: [
+    'Corp Office Reports',
+    'Project Reports',
+    'Watershed Reports',
+    'District Reports'
+  ]
+}
+
+const ReportsPage = () => {
+
+	return (
+		<main>
+      <div className="header-section">
+      <div style={headerContainerStyle} className="p-5">
+      <TextSection
+				containerStyle={containerTextSection}
```

Comment:
@mattmuldoon48 small nitpick but can you tweak the changes across your files so they all use spaces for indents instead of tabs? You can see that there is a mix of space and tab indents which is what is causing these odd looking indents. Thanks. (If you haven't, make sure your IDE is set up to use space indents instead of tabs. I will also add a .editorconfig file to my mock JSON branch that will get merged into develop. Most IDEs will recognize that file and use the settings defined in it.)